src/test/java/com/epam/tc/hw5/ex1/steps/ActionStep.java

```diff
@@ -0,0 +1,28 @@
+package com.epam.tc.hw5.ex1.steps;
+
+import io.cucumber.java.en.When;
+
+@SuppressWarnings("checkstyle:Indentation")
+public class ActionStep extends AbstractStep {
```

Comment:
you don't have to create separated ActionStepS classes for each exercise. fix it.